packages/client/src/components/blocks-workspace/blocks/settings/custom/e-charts/VisualMap.tsx

```diff
@@ -52,7 +52,7 @@ export const VisualMap = observer(
 
         const handleSearch = (e: React.ChangeEvent<HTMLInputElement>) => {
             setSearch(e.target.value);
-            let searchValue = e.target.value;
+            const searchValue = e.target.value;
```

Comment:
do we ever change the value of search value? or does it call handleSearch onChange?